solaris/solaris11/postinstall.sh

```diff
@@ -1,16 +1,17 @@
 #!/bin/sh
 # postinst script for wazuh-agent
-# Wazuh, Inc 2015-2020
+# Wazuh, Inc 2015-2021
```

Comment:
2022. Please replace all years by this new one.

Reply:
Fixed in aa77fe7e88c124e77fbe9aae8edb13afab09a472 and 869642c979b33d843d77f2de525b94fa1ceb5bf0